### PR TITLE
Update homepage.html.twig

### PR DESCRIPTION
**Pull request by 94noni**
following previous PR, with the bloc showcasing hello_controller.js, direct after seeing markdown-controller.js seems not logic (even if stimulus under the hood is ok for both identifiers)

Capture d’écran 2024-04-04 à 22 27 06

### CODE REVIEW

File: ux.symfony.com/templates/main/homepage.html.twig [hunk 1]
```diff
@@ -71,7 +71,7 @@
         </div>
         <div class="col-12 col-md-6  mt-3 mt-md-0">
             <twig:CodeBlock
-                filename="assets/controllers/markdown-controller.js"
+                filename="assets/controllers/markdown_controller.js"
```
Comment (smnandre) [inline ux.symfony.com/templates/main/homepage.html.twig:74]: I think today we used "-controller" as naming convention..

Not sure we should change for only one :|

Comment (94noni) [reply ux.symfony.com/templates/main/homepage.html.twig:74]: What about renaming then the hello-controller?

Comment (smnandre) [reply ux.symfony.com/templates/main/homepage.html.twig:74]: If you think that'll improve the coherence yes for sure!

**Comment by smnandre** [comment]
(same thing here, ok for you @94noni ?)

**Pull request closed without merging**